### PR TITLE
Update pjax-standalone.js

### PR DESCRIPTION
**Pull request by boussou**
Several bugfixes: 
- Title was not taken into account in some situations
- script dynamic loading/execution was not operating on full page content but on the fragment only

### CODE REVIEW

**Comment by villanus** [comment]
Is this safe to merge?

**Comment by thybag** [comment]
I suspect it should be fine, although I've yet to find enough time to give it a proper test.

**Comment by boussou** [comment]
Hi, I am using it, it works.
it depends on whether or not you agree that dynamic execution of scripts should happen even when the <script> tags are outside the container div.

File: pjax-standalone.js [hunk 1]
```diff
@@ -240,7 +240,8 @@
 		// Grab the title if there is one
 		var title = html.getElementsByTagName('title')[0];
 		if(title)
-			document.title = title.innerHTML;
+        //ensure this title is not overwritten in the caller function
+		options.title=document.title = title.innerHTML;
```
Comment (iambacon) [inline pjax-standalone.js:244]: If the `document.title` is being set later on anyway, would it not be better to just set `options.title` here?

**Comment by iambacon** [comment]
+1 for page title bug to be merged.

**Comment by thybag** [comment]
Apologies about not having been able to find any time look at this yet, I've been really busy recently and not had time to look after my GH projects.

For now (since from a quick scan it all looks fine) I've merged the changes in to the develop branch  https://github.com/thybag/PJAX-Standalone/tree/develop - Hopefully when i get time to do some proper testing I'll be able to roll the changes to master.

P.S. Almost forgot to say; thanks for finding & fixing these bugs - its a big help & i really do appreciate it, even if i'm being a bit of a blocker ATM.

**Comment by boussou** [comment]
Hi, back to this project, I am fine with the develop version, thanks.

**Pull request closed without merging**